From 5849b85444cae952be51e63431f32a666dad01c4 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Thu, 3 Nov 2022 11:38:18 +0100
Subject: [PATCH 229/954] feat(stm32mp25-fdts): add sdmmc node for
 STM32MP257F-EV board

Add sdmmc1 node for SD-card.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 73fb687636229e4be9cb7eb504b16d19af9b8995 feat(stm32mp25-fdts): add sdmmc node for STM32MP257F-EV board]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I3bf9253fa5f60eea1725070ae658968687b26f00
---
 fdts/stm32mp257f-ev1.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index b7e92e47c2..58eb1cf235 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -34,3 +34,11 @@
 	pinctrl-0 = <&usart2_pins_a>;
 	status = "okay";
 };
+
+&sdmmc1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&sdmmc1_b4_pins_a>;
+	st,neg-edge;
+	bus-width = <4>;
+	status = "okay";
+};
-- 
2.53.0

